style.css

```diff
@@ -0,0 +1,96 @@
+@font-face {
+  font-family: 'Poppins';
+  src: url(fonts/Poppins-Regular.ttf);
+  font-style: normal;
+}
+
+@font-face {
+  font-family: 'Playfair Display';
+  src: url(fonts/PlayfairDisplay-Regular.ttf);
+  font-style: normal;
+}
+
+body {
+  font-family: 'Poppins', sans-serif;
+  color: #1a2236;
+}
+
+h2 {
+  font-family: 'Playfair Display', sans-serif;
```

Comment:
- [ ] Please use a font family that resembles the original design for this font.